live-site/components/event-schedule-section/event-schedule-tab/EventScheduleTab.tsx

```diff
@@ -12,11 +15,59 @@ import {
   StyledEventScheduleTabContainer
 } from './EventScheduleTab.styles';
 
+const AIRTABLE_BASE_NAME = 'Schedule';
+const AIRTABLE_SCHEDULE_TABLE_NAME = 'schedule';
+const FRI_DATE = new Date(2023, 1, 10);
+const SAT_DATE = new Date(2023, 1, 11);
+const SUN_DATE = new Date(2023, 1, 12);
+
+const renderEventData = (events: any[], activeTab: number): JSX.Element[] => {
+  const toEventItemType = (event: any): EventItemType => {
+    return { id: event.id, ...event.fields };
+  };
+  const convertStringToDate = (event: EventItemType): EventItemType => {
+    console.log(event.time);
+    return { ...event, time: new Date(Date.parse(event.time as string)) };
```

Comment:
i think Date.parse can return `NaN` so it might be worth checking or that here, just a thought